backend/src/main/kotlin/org/loculus/backend/controller/SubmissionController.kt

```diff
@@ -141,6 +141,7 @@ class SubmissionController(
             message = "You can extract at max $MAX_EXTRACTED_SEQUENCE_ENTRIES sequence entries at once.",
```

Comment:
again we should probably add the HttpStatus.NOT_MODIFIED status as an ApiResponse code in the swaggerUI